apps/nextjs-website/src/components/organisms/GitBookContent/components/Hint.tsx

```diff
@@ -1,9 +1,15 @@
-import Alert from '@mui/material/Alert';
+import Alert, { AlertColor } from '@mui/material/Alert';
 import { HintProps } from 'gitbook-docs/markdoc/schema/hint';
 import { ReactNode } from 'react';
 
+const asSeverity = (
+  style: 'info' | 'success' | 'warning' | 'danger'
```

Comment:
I did it to avoid a linting error

Reply:
It was a "Parameter  style  implicitly has an  any  type." error in my IDE. I added and exported the type to use it twice

Reply:
This `style: HintProps<unknown>['style']` should work.

The `unknown` is required because the type is generic, but in this case it doesn't matter what generic is, so you can safely reference the same type for the `style` parameter and you are making sure that any changes on the `HintProps` type will reflect to the component too.